k8s-ingress/README.md

```diff
@@ -0,0 +1,96 @@
+# Linkerd as Kubernetes Ingress Controller
+
+This example application demonstrates the basics of using linkerd as an
+ingress controller. It consists of:
+* `ingress-controller.yaml`: A linkerd config and linkerd deployment listening on `hostPort: 80`.
+* `services.yaml`: Deployment and service definitions for a number of seal-related example services.
+* `ingress.yaml`: An ingress resource that defines routing to those services.
```

Comment:
I think the intro could use a bit more elaboration about what we're trying to accomplish. Something along the lines of:

> The ingress controller in this example routes external requests through linkerd to various backends using the value of the request's Host header. Hence, by specifying different domains we end up with different results. For the purpose of this demo, the domain names are the latin names of various species of pinnipeds (e.g. carnivora.pinniped.odobenidae), and the service responses are the common name of that species (e.g. the walrus).